### ISSUE TITLE
bug: `make build` fails

### ISSUE THREAD

**Issue by MikeSpreitzer** [kcp-dev/edge-mc#116]
### Describe the bug

```
(base) mspreitz@mjs12 edge-mc % make build
hack/verify-go-versions.sh
Wrong go version in .github/workflows/*.yaml, expected 1.18
make: *** [verify-go-versions] Error 1
```

### Steps To Reproduce

`make build` or, even more simply, `make`

### Expected Behaviour

`make` succeeds.

**Comment by clubanderson** [comment]
fixed with #117

**Comment by clubanderson** [comment]
fixed with #118

**Comment by MikeSpreitzer** [comment]
Tested `main` at commit 80870b789cecb6f42d4a4b9efd0eb98a3f98b88d and `make build` works.
Thanks!
